test/Java9andUp/src_current/org/openj9/test/attachAPI/VmIdGetter.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+package org.openj9.test.attachAPI;
+/* This class is separated from the main test because we need two
+ * copies to accommodate old and new versions of ProcessHandle
```

Comment:
@pshipton src_current is for supporting old class library version, which I thought we were trying to move from, instead of build up more test/support for.  Any guidance on this, why we keep adding more test material into this effort?

Reply:
This needs to work on the vmfarm builds, which are stuck on an old (incompatible) class library.

Reply:
This is adding a new test to IBM internal Java 9 b148 builds, correct? I agree its not necessary to add new testing. Existing testing either needs to be fixed or removed to keep the builds green. The b148 builds are obsolete and will be deleted once the equivalent testing is running on OpenJ9.

Reply:
As discussed with the FV team, the tooling doesn't allow us to add the test only for the "latest" (i.e. post b148) JCL, so we will need to maintain 2 copies of some classes (as is done with the StackWalker tests) until we can kill b148.